src/Components/PeoplePage/PeoplePage.js

```diff
@@ -0,0 +1,128 @@
+import React, { Component } from "react";
+import MasonryGallery from "./MasonryComponent.js";
+import FilterDropdown from "./FilterDropdownComponent.js";
+import "./PeoplePage.scss"
+
+class PeoplePage extends Component {
+	constructor() {
+		super();
+		this.state = {
+		   people: [],
+		};
+	}
+
+	setFilter(filterBy){
+
+      const filteredPeople = filterBy === "all"
+         ? this.state.people.map(person => {
+            person.greyedOut = false;
+            return person;
+         })
+         : this.state.people.map(person => {
+            person.greyedOut = !(person.department === filterBy);
+            return person;
+         });
+
+		this.setState({people: filteredPeople});
+	};
+
+	componentWillMount(){
+
+      // Pretend this is from an ajax request or something to get people
+      const people = [
```

Comment:
Removed these from render method as it will get executed everytime we set state which is not needed
Moved these to ComponentWillMount as thats the place where we make Ajax call.